README.md

```diff
@@ -117,9 +122,18 @@ $client_secret = '...'; // This is optional, only needed for impersonation or t
 
 $auth0Api = new Authentication($domain, $client_id, $client_secret); 
```

Comment:
No, they are two different things. I would keep the way to get the oauth client (the Auth0 client) from Authentication for BC